src/axom/inlet/tests/inlet_Inlet.cpp

```diff
@@ -41,10 +42,8 @@ TEST(inlet_Inlet_basic, getTopLevelBools)
   // Define schema
   //
 
-  std::shared_ptr<axom::inlet::Field> currField;
-
   // Check for existing fields
-  currField = inlet->addBool("foo", "foo's description");
+  auto currField = inlet->addBool("foo", "foo's description");
```

Comment:
Minor: This name is a bit misleading now, since `currField` is no longer an `inlet::Field`, it is now an `inlet::VerifiableScalar`, right?